CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Changelog
 ### Fixes
 
 - Fixed delay on loading stake pool list after switching the SMASH server ([PR 2447](https://github.com/input-output-hk/daedalus/pull/2447))
+- Fixed dropdown not appearing in the Stake Pools Settings screen ([PR 2460](https://github.com/input-output-hk/daedalus/pull/2460))
```

Comment:
@daniloprates can you move a log to the top?

Reply:
Fixed in https://github.com/input-output-hk/daedalus/pull/2460/commits/349c8a32837b456bccfd5001e2aaa3538bcf5ccc